src/data/showcases.js

```diff
@@ -1148,6 +1148,15 @@ export const Showcases = [
     source: null,
     tags: ["governance"],
   },
+  {
+    title: "Adastack.io",
+    description:
+      "Comprehenisive Cardano ecosystem explorer: Explore intro guides, wallets, DApps, NFTs, games, governance, Project Catalyst, DAOs, development, sidechains, L2s and more.",
```

Comment:
@katomm I have a question which may be applicable here: why does the Style Guide page spell `dApps` on the [Mermaid diagram](https://developers.cardano.org/docs/portal-style-guide#mermaid) but show `DApps` on the [Editorial Style Guide table](https://developers.cardano.org/docs/portal-style-guide#editorial-style-guide)?

Reply:
I don't know, we just need to agree.

Reply:
Grammatically it should be `dApp` or `dApps` since there's no reason to capitalise the word itself (a common noun, even though specialised)... as opposed to `DRep` which is a specific title (not just a "job" which would be a common noun & not capitalised).  I'll put in a PR soon to update the Style Guide and we can agree upon it there & will correct it here in the meantime (and will change it in that PR if the consensus goes the other way).

Reply:
p.s. I am seeing now that [the Wikipedia page](https://en.wikipedia.org/wiki/Decentralized_application) has it capitalised, although listing both spellings as correct.  So let's leave it as it is here... and though I'd hate to put such a statement in an official "Style Guide" I think @katomm it should therefore say that either spelling is acceptable. 🤔